sync/syncer.go

```diff
@@ -222,14 +227,14 @@ func (s *Syncer) Run(ctx context.Context) error {
 // logCountsForOpTypes tries to aggregate the various device "op_type"
 // attributes so they can be logged.
 func logCountsForOpTypes(isFetch bool, devices []godep.Device) []interface{} {
-	opTypes := map[string]int{"added": 0, "modified": 0, "deleted": 0, "other": 0}
+	// Empty op_type "" is returned by FetchDeviceRequest.
```

Comment:
logging a separate op_type for the empty case is just the count of all the devices for a fetch operation — which is already logged above. I could be convinced that this is more explicit (and thus better), however if you felt strongly about it.